spec/helpers/download_presence_spec.rb

```diff
@@ -21,6 +21,9 @@
     download = LanguagePack::Helpers::DownloadPresence.new(
       multi_arch_stacks: ["heroku-24"],
       file_name: 'ruby-3.0.5.tgz',
+      # Heroku 20 is not longer supported however heroku-22 and heroku-24
```

Comment:
Does this test make sense to keep if you have to use the `heroku-20` stack for it to be functional?

Reply:
I need at least 2 stacks with versions where the binaries have different versions. That won't be true again until heroku-26 is released. Then we can update the test:

```ruby
  it "handles multi-arch transitions for files that do not exist" do
    download = LanguagePack::Helpers::DownloadPresence.new(
      multi_arch_stacks: ["heroku-24"],
      file_name: 'ruby-3.0.5.tgz',
      # Heroku 20 is not longer supported however heroku-22 and heroku-24
      # have identical Ruby versions supported, this test can be updated to
      # use heroku-24 and heroku-26 (when that stack is released)
```